sycl/test/CMakeLists.txt

```diff
@@ -75,6 +75,34 @@ if(SYCL_BUILD_PI_CUDA)
   add_custom_target(check-sycl-cuda)
   add_dependencies(check-sycl-cuda check-sycl-ptx)
   add_dependencies(check-sycl check-sycl-cuda)
+endif()
+
+if(SYCL_BUILD_PI_ROCM)
+  add_custom_target(check-sycl-rocm)
+  if("${SYCL_BUILD_PI_ROCM_PLATFORM}" STREQUAL "NVIDIA")
```

Comment:
Very minor comment. 

See the discussion in https://github.com/intel/llvm/pull/4069. In case we need to support other spellings for `SYCL_BUILD_PI_ROCM_PLATFORM`, we might need to make sure it's all capital.

Reply:
Yeah that's probably a good idea, however there's already a bunch of other places that do this comparison. So I'd suggest to keep it like that in this patch and then once it's merged I can put up a new PR updating all of the uses of this to be case-insensitive.